rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -227,4 +228,21 @@ get_c_vector_string(const std::vector<std::string> & strings_in)
   return cstrings;
 }
 
+std::pair<int32_t, uint32_t>
+convert_rcl_time_to_sec_nanos(const rcl_time_point_value_t & time_point)
```

Comment:
How about having this as free-function in `rclcpp` scope in `time.cpp/hpp` since it is obviously related to time conversion? besides, we can also replace here with it?

https://github.com/ros2/rclcpp/blob/c10764f4329f8c1494b68ff60ce55a3c74c8de9e/rclcpp/src/rclcpp/time.cpp#L94-L102

```suggestion
builtin_interfaces::msg::Time
convert_rcl_time_to_sec_nanos(const rcl_time_point_value_t & time_point)
```

Reply:
LGTM and makes much sense, push a newer version.